pages/super_search_page.py

```diff
@@ -130,10 +129,14 @@ def results_facet(self):
     def is_column_in_list(self, column_name):
         return column_name in [column.column_name for column in self.columns]
 
-    class SearchResultHeader(Page):
+    class SearchResultHeader(object):
 
         _table_header_name_locator = (By.CSS_SELECTOR, '#reports-list thead th')
 
+        def __init__(self, selenium, timeout):
+            self.selenium = selenium
+            self.timeout = timeout
```

Comment:
Good advice. I'll add this, will take until next week though. Too much EOQ work on my plate right now